tests/Fixtures/TestBundle/State/OperationResourceProcessor.php

```diff
@@ -56,7 +56,7 @@ private function remove($data): void
         $manager->flush();
     }
 
-    public function process(mixed $data, Operation $operation, array $uriVariables = [], array $context = [])
+    public function process(object $data, Operation $operation, array $uriVariables = [], array $context = []): ?object
```

Comment:
To be consistent with `src/Doctrine/Common/State/RemoveProcessor.php`, you should return null in case of DeleteOperationInterface.